common/components/pagination/template.test.js

```diff
@@ -184,60 +184,72 @@ describe('Pagination component', function () {
       expect($itemLink.attr('href')).to.equal('/page-3')
     })
   })
-})
 
-context('with items with class', function () {
-  let $, $component, items
+  context('with items with class', function () {
```

Comment:
The indentation of these tests was previously incorrect, they were sat outside the containing block that describes this component. That's why this change is showing some whitespace changes.